AzureMonitorAgent/agent.py

```diff
@@ -1224,6 +1224,29 @@ def generate_localsyslog_configs():
         if features is not None and "useSyslogTcp" in features:            
             useSyslogTcp = features.get("useSyslogTcp")    
     
+    # always use syslog tcp port, unless 
+    # - the distro is Red Hat based and doesn't have semanage
+    #   these distros seem to have SELinux on by default and we shouldn't be installing semanage ourselves
+    distro, version = find_vm_distro('Enable')
+    for name in ['ubuntu', 'debian']:
```

Comment:
Instead of being dependent on distro type is there a way to check se system setting instead? We will also have cases when it is one of the distroes where selinux is not enabled by default but it is enabled by customer and where the port will fail to be enabled.